res/layout/share_public_link_item.xml

```diff
@@ -64,7 +64,7 @@
             android:layout_height="wrap_content"
             android:layout_gravity="center_vertical"
             android:padding="@dimen/standard_half_padding"
-            android:src="@drawable/ic_navigate_next" />
+            android:src="@drawable/ic_lead_pencil_grey" />
```

Comment:
@theScrabi This pencil looks like it comes from the pre-Material time in Android. You might want to use [this icon](https://material.io/tools/icons/?icon=edit&style=baseline) as svg, not png.

Reply:
I thought it would look a little slim with the new Icon. but yes i can give it a shot.